tests/cisco/test_cisco_switch_protocol.py

```diff
@@ -194,7 +194,7 @@ def test_show_vlan_brief(self, t):
         t.readln("")
         t.readln("VLAN Name                             Status    Ports")
         t.readln("---- -------------------------------- --------- -------------------------------")
-        t.readln("1    default                          active    Fa0/2, Fa0/3, Fa0/4")
+        t.readln("1    default                          active    Fa0/2, Fa0/3, Fa0/4, Fa0/5, Fa0/6, Fa0/7, Fa0/8, Fa0/9, Fa0/10, Fa0/11, Fa0/12")
```

Comment:
Why an issue? Why introduce an inconsistency?

To my knowledge we already wrap to 80 chars everywhere